### DISCUSSION TITLE
Continue use of GH or move to a more politically friendly platform.

### DISCUSSION THREAD

**Discussion by royemosby** [nditcommunity/ndit-website#48] [Technology]
Should we continue the use of Github or move to another version control platform? What would be the pros and cons compared to the breadth of services GH offers?

Some alternatives?

- [https://git.gay/](https://git.gay/)
- [Codeberg](https://codeberg.org)

**Comment by marcusolsson** [comment]
Can I ask what makes GitHub a politically unfriendly platform? I'm aware of the news of their contract with ICE, but not sure if it's still active.

It would depend on whether we're looking to host the website on the platform as well. Both GitHub and [GitLab](https://gitlab.com) have free hosting with their Pages offering.

Otherwise, if we want to host it on Netlify or similar, I find the GitHub integrations really helpful.

**Reply by royemosby** [reply]
This is not my claim - it's just another conversation that I moved from the website channel in Slack. Most of the issues that some have with GH is its ownership by Microsoft and the use of repo content to train up Copilot.

**Reply by royemosby** [reply]
Not related to this specific topic, I think Netlify would be a good idea. Since we'll have multiple people working with this, it would be nice to have branch deploys for sandboxes or staging. I already want that now as I bootstrap the SSG

**Comment by hayleyw7** [comment]
I've never heard about it. Thanks for sharing. I hate to say it, but I do still lean to keeping with GitHub so that there's the lowest possible barrier to entry for contributors.

And with it being the industry standard and where future employers look to see activity, I think people are most likely to help where they can get more public credit.

**Reply by royemosby** [reply]
I am with you mostly due to "to many logins" fatigue. I'm hesitant to establish yet another account + learn another platform. Also, since our community's focus is on its people and not specifically technologic or foss-related activism, idealogical hosting decisions shouldn't take precedence over getting something out there to spread the word of our community.

**Comment by hayleyw7** [comment]
I'm down with that!
